oci_java_image/BUILD.bazel

```diff
@@ -0,0 +1,45 @@
+load("@rules_oci//oci:defs.bzl", "oci_image", "oci_tarball", "structure_test")
```

Comment:
old version of rules_oci?

Reply:
still the latest release :)
We should cut one before the actual RC and rollout here and other places to ensure we didn't miss something obvious